bcachefs-hooks

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/ash
+
+run_hook() {
+   if bcachefs unlock -c $root >/dev/null 2>&1; then
+    	echo "Unlocking $root:"
+    
+    	while true; do
+    		bcachefs unlock $root && break
+    	done
+    fi
+
+    echo "running fsck.bcachefs on $root:"
+    fsck.bcachefs $root
```

Comment:
won't this be done automatically anyway?

Reply:
I've had issues with the default `fsck` hook. It ends up running `fsck.ext2` for some reason, and since my root partition is a bcachefs volume not a ext2 volume it fails. I'm not sure why it does that. I can try regenerating my initramfs and see if that changes anything since I updated `bcache-tools` to the latest git version. It should be handled by the `fsck` and technically doesn't need to be in there.